### DISCUSSION TITLE
Show git HEAD info on the deployment page via CLI

### DISCUSSION THREAD

**Discussion by jaeseokk** [vercel/vercel#9160] [Help] [answered]
When I deploy via CLI, Is there a way to show the git HEAD info on the deployment page?

<img width="1170" alt="image" src="https://user-images.githubusercontent.com/12602843/210492512-653d1742-d87a-4c6d-a66c-01ef1ffa5c9a.png">

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @jaeseokk! Are you using the latest version of Vercel CLI (28.10.2) or an older version? Some older versions may not have support for pulling in git metadata.

Another problem I have seen recently is that the CLI is not pulling in git metadata when no remote GitHub, GitLab, or Bitbucket url is detected for the local repository. The CLI team has a ticket open to address that issue.

**Reply by jaeseokk** [reply]
Although using the latest version of Vercel CLI, git metadata has not been shown up. but I found that HEAD commit information does not show up if gitDirty="1".
After deleting the local changes and deploying it, the git metadata shows up well.

thanks for your help!
